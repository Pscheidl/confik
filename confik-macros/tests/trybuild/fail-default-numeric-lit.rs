#[derive(confik::Configuration)]
struct Config {
    #[confik(default = 123)]
    _param: String,
}

fn main() {}
