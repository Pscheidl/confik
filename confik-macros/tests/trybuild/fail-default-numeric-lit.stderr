error: Unexpected literal type `int`
 --> tests/trybuild/fail-default-numeric-lit.rs:3:24
  |
3 |     #[confik(default = 123)]
  |                        ^^^
